food.go

```diff
@@ -246,7 +246,14 @@ func (f *Food) Link(pkg *Package) error {
 	barrelDir := filepath.Join(home.Barrel(), f.Name, f.Version)
 	for _, r := range pkg.Resources {
 		// TODO: run this in parallel
-		destPath := filepath.Join(home.HomePrefix, r.InstallPath)
+
+		// We assume every Food's InstallPath begins with `bin/` (`bin\\` on Windows)
+		installPath, err := filepath.Rel("bin", r.InstallPath)
+		if err != nil {
+			return err
+		}
+
+		destPath := filepath.Join(home.BinPath(), installPath)
```

Comment:
A potentially alternative implementation to this would be introduce something like

```
homeprefixpath = lazypath.LazyPath{
	EnvironmentVariable: DefaultHomePrefixEnvVar,
	DefaultFn:           defaultHomePrefix,
}


// HomePrefixPath is the path under which home and bin direcotries should reside.
func HomePrefixPath() string {
	return homeprefixpath.Path("")
}
```

and use

```
destPath := filepath.Join(home.HomePrefixPath(), r.InstallPath)
```